kind/actions/kind.go

```diff
@@ -236,13 +236,18 @@ func ListControlPlanes(clusterName string) ([]nodes.Node, error) {
 		fmt.Sprintf("label=%s=%s", constants.NodeRoleKey, constants.ControlPlaneNodeRoleValue))
 }
 
-// getLoadBalancerPort returns the port on the host on which the APIServer is exposed
-func getLoadBalancerPort(allNodes []nodes.Node) (int32, error) {
+// GetLoadBalancerHostAndPort returns the port on the host on which the APIServer is exposed
+func GetLoadBalancerHostAndPort(allNodes []nodes.Node) (string, int32, error) {
```

Comment:
This gets the docker IP of the load balancer container. We use this to write the Docker IP into the kubeadm config for the node-ref controller